### PR TITLE
added basic junit 5 support as new java 11 profile

### PR DESCRIPTION
**Pull request by sschlick**
added basic junit 5 support as new java 11 profile

### CODE REVIEW

File: PVEntityGenerator/xml/PVEntityGenerator.xml [hunk 1]
```diff
@@ -7,6 +10,7 @@
     <platform name="Java2" definition-dir="Platform_Java2"/>
     <platform name="Java5" definition-dir="Platform_Java5"/>
     <platform name="Java5JPA" definition-dir="Platform_Java5JPA"/>
+	  <platform name="Java11" definition-dir="Platform_Java11"/>
```
Comment (stefanseifert) [inline PVEntityGenerator/xml/PVEntityGenerator.xml]: is this new platform derived from Platform_Java5 or Platform_Java5JPA? in case of the latter, i would prefer a name like "Platform_Java11JPA" to reflect this.

cosmetic: please fix indentation.

File: PVEntityGenerator/xml/PVEntityGenerator.xml [hunk 2]
```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!-- $URL$ -->
```
Comment (stefanseifert) [inline PVEntityGenerator/xml/PVEntityGenerator.xml]: please do not add this (back), it's SVN-specific

File: PVEntityGenerator/PVEntityGenerator.csproj [hunk 3]
```diff
@@ -25,7 +25,7 @@
     </FileUpgradeFlags>
     <UpgradeBackupLocation>
     </UpgradeBackupLocation>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
```
Comment (stefanseifert) [inline PVEntityGenerator/PVEntityGenerator.csproj]: it should not be required to update the dependency here in this PR - if required this should go to a separate PR.

File: PVEntityGenerator/app.config [hunk 4]
```diff
@@ -17,4 +17,4 @@
       <remove invariant="MySql.Data.MySqlClient"/>
       <add name="MySQL Data Provider" invariant="MySql.Data.MySqlClient" description=".Net Framework Data Provider for MySQL" type="MySql.Data.MySqlClient.MySqlClientFactory, MySql.Data, Version=6.9.8.0, Culture=neutral, PublicKeyToken=c5687fc88969c44d"/>
     </DbProviderFactories>
-  </system.data><startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1"/></startup></configuration>
+  </system.data><startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.8"/></startup></configuration>
```
Comment (stefanseifert) [inline PVEntityGenerator/app.config]: it should not be required to update the dependency here in this PR - if required this should go to a separate PR.

File: PVEntityGenerator/PVEntityGenerator.csproj [hunk 5]
```diff
@@ -81,6 +81,29 @@
     <PlatformTarget>x86</PlatformTarget>
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x86'">
```
Comment (stefanseifert) [inline PVEntityGenerator/PVEntityGenerator.csproj]: this new block should not be required for this PR?

File: PVEntityGenerator/PVEntityGenerator.sln [hunk 6]
```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26430.14
+# Visual Studio Version 17
+VisualStudioVersion = 17.2.32526.322
```
Comment (stefanseifert) [inline PVEntityGenerator/PVEntityGenerator.sln]: it should not be required to update the dependency here in this PR - if required this should go to a separate PR.

**Review by stefanseifert — CHANGES_REQUESTED** [review]

File: PVEntityGenerator/xml/PVEntityGenerator.xml (hunk 2 above)
Comment (sschlick) [reply PVEntityGenerator/xml/PVEntityGenerator.xml]: ok, removed

File: PVEntityGenerator/PVEntityGenerator.csproj (hunk 5 above)
Comment (sschlick) [reply PVEntityGenerator/PVEntityGenerator.csproj]: ok, removed

File: PVEntityGenerator/xml/PVEntityGenerator.xml (hunk 1 above)
Comment (stefanseifert) [reply PVEntityGenerator/xml/PVEntityGenerator.xml]: please also rename the platform itself (similar to Java5JPA), and ensure the folder names and references in the XSLT to it are updated as well.

Comment (sschlick) [reply PVEntityGenerator/xml/PVEntityGenerator.xml]: ok, renamed to Java11JPA/Platform_Java11JPA

**Review by stefanseifert — APPROVED** [review]
lgtm, thanks!
